crates/dojo-world/src/manifest.rs

```diff
@@ -110,6 +111,42 @@ pub struct ComputedValueEntrypoint {
     pub model: Option<String>,
 }
 
+/// Format of the ABI into the manifest.
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum AbiFormat {
+    /// Only a relative path to the ABI file is stored.
+    Path(Utf8PathBuf),
+    /// The full ABI is embedded.
+    Embed(Vec<AbiEntry>),
+}
+
+impl PartialEq for AbiFormat {
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
is this implementation required? i couldn't find where this trait is used.

Reply:
I mostly derived it because `DojoModel` and other related types all implement it.
If we don't use this at all, I can actually try to remove this trait everywhere. :)